prisma/schema.prisma

```diff
@@ -208,7 +214,9 @@ model Goal {
   priority                String?
   priorityExact           Priority?             @relation(fields: [priorityId], references: [id])
   priorityId              Int?
-  estimate                EstimateToGoal[]
+  estimate                DateTime?
+  estimateType            DateType?
+  legacyEstimate          EstimateToGoal[]
```

Comment:
I think it must be vise versa. We don't need to rename current fields. Add new only.